src/Kliva/Controls/ActivityFeedControl.xaml

```diff
@@ -17,6 +17,24 @@
 
     <Grid>
         <VisualStateManager.VisualStateGroups>
+            <VisualStateGroup x:Name="AdaptiveStates" CurrentStateChanged="AdaptiveStates_CurrentStateChanged">
```

Comment:
I'm assuming you mean ActivityDetailControl? I don't think any changes need to be made there as we still want the differentiated behavior for full-page versus side-to-side.